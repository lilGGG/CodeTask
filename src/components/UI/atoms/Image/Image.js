import styled from 'styled-components';
// import { configure } from "enzyme";
// import Adapter from "enzyme-adapter-react-16";

// configure({ adapter: new Adapter() });

// import Enzyme from 'enzyme';
// import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

// Enzyme.configure({ adapter: new Adapter() });

export const ImageShows =  styled.img`
    width: 200px;
    height: auto;
`;


function Image({
   SrcImage
}) {
  return (
      <>
       <ImageShows src= {SrcImage} />
      </>
  ) 
}

export default Image;
