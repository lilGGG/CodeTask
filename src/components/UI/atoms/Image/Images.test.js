import React from "react";
import Image from "./Image";
import ReactDom from "react-dom";
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import { create, act } from "react-test-renderer";
import { shallow, mount, render } from "enzyme";

describe('Should Preferences', () => {

    it('renders correctly', () => {
      const tree = renderer.create(<Image SrcImage="https://static.tvmaze.com/uploads/images/medium_portrait/24/60941.jpg"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });

    // it("should render content", () => {
    //     act(() => {
    //       root = create(<Image />);
    //     });
    //     expect(root.toJSON()).toMatchInlineSnapshot();
    //   });

    //   it("Should test component", () => {
    //     const wrapper = shallow(<Image SrcImage="https://static.tvmaze.com/uploads/images/medium_portrait/24/60941.jpg"/>);
    //     expect(wrapper).toMatchSnapshot();
    //   });
      
    // it('renders correctly', () => {
    //     const tree = renderer.create(<Image />).toJSON();
    //     expect(tree).toMatchInlineSnapshot();
    //   });
  });
  
