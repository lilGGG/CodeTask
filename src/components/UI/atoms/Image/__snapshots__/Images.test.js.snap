// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Preferences renders correctly 1`] = `
<img
  className="sc-bczRLJ fASZzU"
  src="https://static.tvmaze.com/uploads/images/medium_portrait/24/60941.jpg"
/>
`;
